src/backend/executor/bridge.ts

```diff
@@ -130,73 +146,70 @@ export class ExecutorBridge implements Disposable {
         }
 
         const filePaths = files.length
-            ? `--file ${files.map((uri) => `"${uri.fsPath}"`).join(' ')}`
-            : '';
+            ? ['--file', ...files.map((uri) => uri.fsPath)]
+            : [];
 
-        return [
-            `${ccPath} analyze`,
-            `"${ccCompileCmd}"`,
-            `--output "${ccFolder}"`,
-            `${ccThreads ? '-j ' + ccThreads : ''}`,
-            `${ccArguments}`,
-            `${filePaths}`,
-        ].join(' ');
+        const args = [
+            'analyze', ccCompileCmd,
+            '--output', ccFolder,
+        ];
+
+        if (ccThreads) {
+            args.push('-j', ccThreads);
+        }
+
+        args.push(...ccArguments, ...filePaths);
+
+        return args;
     }
 
-    public getLogCmdLine(): string | undefined {
+    public getLogCmdArgs(): string[] | undefined {
         if (!workspace.workspaceFolders?.length) {
             return undefined;
         }
 
         const workspaceFolder = workspace.workspaceFolders[0].uri.fsPath;
-
-        const ccPath = getConfigAndReplaceVariables('codechecker.executor', 'executablePath')
-            ?? 'CodeChecker';
         const ccFolder = getConfigAndReplaceVariables('codechecker.backend', 'outputFolder')
             ?? path.join(workspaceFolder, '.codechecker');
 
         // Use a predefined path here
         const ccCompileCmd = path.join(ccFolder, 'compile_commands.json');
 
         return [
-            `${ccPath} log`,
-            `--output "${ccCompileCmd}"`,
-            '--build "make"'
-        ].join(' ');
+            'log',
+            '--output', ccCompileCmd,
+            '--build', 'make'
+        ];
     }
 
-    public getParseCmdLine(...files: Uri[]): string | undefined {
+    public getParseCmdArgs(...files: Uri[]): string[] | undefined {
         if (!workspace.workspaceFolders?.length) {
             return undefined;
         }
 
         const workspaceFolder = workspace.workspaceFolders[0].uri.fsPath;
 
-        const ccPath = getConfigAndReplaceVariables('codechecker.executor', 'executablePath')
-            ?? 'CodeChecker';
         const ccFolder = getConfigAndReplaceVariables('codechecker.backend', 'outputFolder')
             ?? path.join(workspaceFolder, '.codechecker');
 
+
         const filePaths = files.length
-            ? `--file ${files.map((uri) => `"${uri.fsPath}"`).join(' ')}`
-            : '';
+            ? ['--file', ...files.map((uri) => uri.fsPath)]
+            : [];
```

Comment:
Could the parse command contain an option specifying the skiplist file?
`--file` and `--skip` options are mutually exclusive options AFAIK.

Reply:
Additionally to this, the file option will parse the file path and do a lookup in the compdb.json to find the corresponding entry for that TU. However, if there are symbolic links, the two might mismatch. It happens for me at least, and it basically a blocker for analyzing any files. However, we might need to fix this on the CodeChecker side. Im not sure.

Reply:
Indeed they are, created an issue for this: #74.

Reply:
I'm not sure if it's even possible to solve the second issue on our side without resolving all paths in the comp. db.

Reply:
Yea, well. I saw once that the plugin copied the json. That being said, you can lookup all entries there and resolve than replace the occurrances. Basically patching the local copy db, where the canonical representation would be the resolved paths to the given TU. Keep in mind that the original db cannot be modified, since that might be generated by cmake for example. Alternatively you should document this behavior and even better, have a foolproof test in code and a popup-hint that there is a mismatch. You should 'log' and create a db.json using the resolved path as a working directory. BTW the CodeChecker ldlogger should be able to workaround this, but thats a different topic.